### ISSUE TITLE
EXTENSION_NO_INTRA_EMPHASIS is ignored

### ISSUE THREAD

**Issue by jecolon** [russross/blackfriday#178]
Input
```
some para_graph with _soft_ break 
inside it.
```

Output
```
<p>some para<em>graph with _soft</em> break
inside it.</p>
```

Code
```
	extensions := 0
	extensions |= blackfriday.EXTENSION_NO_INTRA_EMPHASIS
	extensions |= blackfriday.EXTENSION_TABLES
	extensions |= blackfriday.EXTENSION_FENCED_CODE
	extensions |= blackfriday.EXTENSION_AUTOLINK
	extensions |= blackfriday.EXTENSION_STRIKETHROUGH
	extensions |= blackfriday.EXTENSION_SPACE_HEADERS
	htmlFlags := 0
	htmlFlags |= blackfriday.HTML_USE_SMARTYPANTS
	htmlFlags |= blackfriday.HTML_SMARTYPANTS_FRACTIONS
	renderer := blackfriday.HtmlRenderer(htmlFlags, "", "")
```

**Comment by shurcooL** [comment]
I can reproduce this issue.

**Comment by jecolon** [comment]
Thanks for the response. It's weird that this happens when I POST the markdown via the Chrome PostMan RESTClient extension, but if I post it via Javascript (AngularJS) it works fine. So I assume it's something to do with PostMan https://chrome.google.com/webstore/detail/postman-rest-client/fdmmgilgnpjigdojojpjoooidkmcomcm?hl=en

Once again, thanks! BTW, great job on this package; it's awesome.

**Comment by shurcooL** [comment]
You've closed this issue, but I don't think it's resolved.

Here's a reproduce case. https://github.com/shurcooL/play/blob/master/125/main.go It occurs with the latest blackfriday.

Reopening since this is a valid unresolved issue.

**Comment by rtfb** [comment]
I can no longer reproduce this. Bisecting says it was fixed by cc3cc10. I'll put up some tests to nail down this functionality and then close.

**Comment by shurcooL** [comment]
Confirmed, it's no longer an issue on latest `master`.
